### ISSUE TITLE
water_still.png Crashes 

### ISSUE THREAD

**Issue by mandm635** [MrCrayfish/ModelCreator#74]
Using the current latest release(pre4), it seems to always crash when attempting to apply water_still.png. I am able to apply lava_still.png with no problem, and even get the animation? This would be nice to see fixed soon!

**Comment by sqidopus** [comment]
Yeah i got the same thing. If you run the program with command prompt then it gives you an Arithmetic Exception and then a few lines of stack trace. An easy fix is to throw the problem code in a try and catch statement which isnt too hard

**Comment by AndersBillLinden** [comment]
This is how I tried to produce an error, from an empty project, creating a cube, go to the "faces" tab, click "Image", click "import", navigate to water_still.png, click "open". The texture shows. This will finish the test. It has succeeded using water_still.png from 1.8.8, 1.8.9 and 1.9 (the two first being the same).

**Comment by MrCrayfish** [comment]
I believe this was fixed ages ago, just haven't pushed a new pre out.

**Comment by AndersBillLinden** [comment]
I used mc-pre4.exe when testing.
